rgit.rb

```diff
@@ -35,16 +34,50 @@ def add(*pathspecs)
     headRepo.add(matched_files)
   end
 
-  desc "git", "Calls vanilla git with your args"
+  @init_descriptions = {
+      quiet:
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.